### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by jtaleric**
### Description 
Tisk tisk we are missing the Apache 2 License

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (afcollins) [inline LICENSE:189]: ?

Comment (jtaleric) [reply LICENSE:189]: yeah, this is just the boiler plate license, I can add the year and Cloud-Bulldozer org.

**Comment by jtaleric** [comment]
cc @rsevilla87 @mohit-sheth @venkataanil @afcollins 

Since this is introducing a License we need consensus.

**Review by afcollins — APPROVED** [review]

**Review by rsevilla87 — APPROVED** [review]
lgtm

**Comment by rsevilla87** [comment]
cc: @morenod

**Pull request merged**